github.css

```diff
@@ -0,0 +1,99 @@
+/*
+
+github.com style (c) Vasily Polovnyov <vast@whiteants.net>
```

Comment:
Selenium doesn’t require third-party libraries to carry the same license, and a BSD license is obviously fine to include.

Can you include a reference to it in [attr.html](https://github.com/SeleniumHQ/docs/blob/gh-pages/attr.html)?